website/routes.py

```diff
@@ -227,6 +228,20 @@ def make_url_map(app):
         ),
 
         Rule('/news/', 'get', {}, OsfWebRenderer('public/pages/news.mako')),
+
+        Rule(
+            '/prereg/',
+            'get',
+            prereg.prereg_landing_page,
+            OsfWebRenderer('prereg_landing_page.mako', trust=False)
+        ),
+
+        Rule(
+            '/api/v1/prereg/draft_registrations/',
```

Comment:
A long time ago, yeah, at least for the API route. And that should be more ephemeral than many, hopefully.

The `/prereg/` route is new. I think that ones fine, insofar as, since it's marketing related. The one thing that will need to be done is that we'll need to blacklist the term for our guid picker, since we will eventually have 7 character guids.